LongevityWorldCup.Website/wwwroot/partials/leaderboard-content.html

```diff
@@ -603,6 +609,8 @@ <h3>Proofs</h3>
 <script>
     let athleteResults = [];
 
+    const currentSeasonStartDate = new Date('2025-01-01');
```

Comment:
Since this is hardcoded, we need to keep it in mind to update it every year. How about making it dynamic?

```
const currentSeasonStartDate = new Date(new Date().getFullYear(), 0, 1);
```